index.html

```diff
@@ -95,6 +95,25 @@ <h1>Welcome</h1>
 
           <h2>News</h2>
 
+          <div class="news_item">
+            <h3>May 8, 2019</h3>
+            <p>OpenVDB 6.1.0 is now available to
+               <a href="./download/" target="_blank">download</a>.</p>
+            <p>Highlights of this release:
+            <ul>
+                <li>a significantly expanded and improved CMake build system</li>
+                <li>new <a href="./documentation/doxygen/build.html" target="_blank">
+                    build process</a> and
+                    <a href="./documentation/doxygen/dependencies.html" target="_blank">
+                    build dependency</a> documentation</li>
+                <li>point data grid performance improvements</li>
+                <li>improvements and bug fixes to the Houdini toolkit</li>
+                <li>I/O and metadata support for point data grids in the Python module.</li>
+            </ul>
+            <p>See the <a href="./documentation/doxygen/changes.html#v6_1_0_changes"
+               target="_blank">release notes</a> for a complete list of changes.</p>
```

Comment:
Perhaps worth a quick note here to mention about the new H16.5 minimum requirement?

Reply:
Good idea.